src/main/scala/scalismo/faces/sampling/face/proposals/GaussianRotationProposal.scala

```diff
@@ -50,11 +50,11 @@ case class GaussianRotationProposal(axis: Vector[_3D], sdev: Double)(implicit rn
       val dy = to.yaw - from.yaw
       val dr = to.roll - from.roll
       val rot = Rotation3D.fromEulerXYZ(dp, dy, dr)
-      if (math.abs(rot.phi) > 1e-5 && (rot.axis dot axis) < 1.0 - 1e-4)
+      if (math.abs(rot.phi) > 1e-5 && math.abs(rot.axis dot axis) < 1.0 - 1e-4)
```

Comment:
As far as I see it, here we make sure that the rotation has to be around `axis` by assigning NegativeInfinity to where it is not. 
Consider this example: rot.axis is (1,0,0) and axis is (3,0,0) they are parallel, but the dot product gives us 3. Shouldn't the normalized axis be used for this calculation?
`rot.axis dot normAxis`

Reply:
Maybe we should in addition to the absolute value add the normalized axis.... However, the normalization only fixes the length, which should in our software be no problem. Using these proposals we usually start with a normalized axis and so they should be normalized for all our current proposals up to numerical issues.

The current occurring problem is, that the axis can point in the opposite direction. This you will not fix with a normalization. In those cases, the dot product is -1 and the second part of the if condition prevents from handling the rotation as stemming from this proposal.

Reply:
It is not made sure that a normalized axis is supplied to the constructor, therefore, in my opinion the normalized axis should be used in this line, I know that it does not fix the bug, but this is a nice opportunity to also fix this small inconsistency.